R/fill.R

```diff
@@ -3,14 +3,15 @@ NULL
 
 #' Fill in missing values.
 #'
-#' Fills missing values in using the previous entry. This is useful in the
+#' Fills missing values in selected columns using the previous entry. This is useful in the
 #' common output format where values are not repeated, they're recorded
 #' each time they change.
 #'
 #' Missing values are replaced in atomic vectors; `NULL`s are replaced
 #' in list.
 #'
 #' @inheritParams gather
+#' @param ... A selection of columns. If empty, nothing happens. You can supply bare variable names, select all variables between x and z with x:z, exclude y with -y. For more selection options, see the [dplyr::select()] documentation.
```

Comment:
This is the right fix, but can you please wrap these lines at ~80 cols? Thanks!

Reply:
Fixes #485

Reply:
Thanks